wlroots 0.12.0 is out-of-date and has no other consumers. Keep it
private to phoc package for now.

--- subprojects/wlroots/include/meson.build.orig	2021-08-20 08:28:34 UTC
+++ subprojects/wlroots/include/meson.build
@@ -8,7 +8,9 @@ if conf_data.get('WLR_HAS_XWAYLAND', 0) != 1
 	exclude_files += 'xwayland.h'
 endif
 
+if not meson.is_subproject()
 install_subdir('wlr',
 	install_dir: get_option('includedir'),
 	exclude_files: exclude_files,
 )
+endif
--- subprojects/wlroots/include/wlr/meson.build.orig	2021-08-20 08:28:34 UTC
+++ subprojects/wlroots/include/wlr/meson.build
@@ -16,4 +16,6 @@ ver_h = configure_file(
 	configuration: version_data,
 )
 
+if not meson.is_subproject()
 install_headers(conf_h, ver_h, subdir: 'wlr')
+endif
--- subprojects/wlroots/meson.build.orig	2021-09-12 16:57:08 UTC
+++ subprojects/wlroots/meson.build
@@ -147,7 +147,7 @@ lib_wlr = library(
 	soversion: soversion,
 	dependencies: wlr_deps,
 	include_directories: [wlr_inc, proto_inc],
-	install: true,
+	install: not meson.is_subproject(),
 	link_args: symbols_flag,
 	link_depends: symbols_file,
 )
